tests/phpunit/src/Commands/Api/ApiCommandTest.php

```diff
@@ -400,4 +400,60 @@ public function testGetApplicationUuidFromBltYml(): void {
     $this->fs->remove($bltConfigFilePath);
   }
 
+  /**
+   * Test of deletion of the user from organization by user uuid.
+   */
+  public function testOrganizationMemberDeleteByUserUuid(): void {
+    ClearCacheCommand::clearCaches();
+    $org_id = 'bfafd31a-83a6-4257-b0ec-afdeff83117a';
+    $member_uuid = '26c4af83-545b-45cb-b165-d537adc9e0b4';
+
+    $response = $this->getMockResponseFromSpec('/organizations/{organizationUuid}/members/{userUuid}', 'delete', 200);
```

Comment:
Note to self, I'll try to replace this with the new mockResponse method (may require some tweaks to that method)